resource_processor/.env.sample

```diff
@@ -0,0 +1,16 @@
+PYTHONPATH="."
+AZURE_CLIENT_ID=__CHANGE_ME__
```

Comment:
Yeah its horrible - Im currently reworking the resource processor debugging experience so it'll just use tre.env values. However I'm certainly not doing a consolidation of the others in my PR xD this will have to be dealt with separately